osl/sandbox.go

```diff
@@ -32,6 +32,12 @@ type Sandbox interface {
 	// Unset the previously set default IPv6 gateway in the sandbox
 	UnsetGatewayIPv6() error
 
+	// AddVirtualIP adds the passed IP address to the sandbox loopback interface
+	AddVirtualIP(ip *net.IPNet) error
+
+	// RemoveVirtualIP removes the passes IP address from the sandbox  loopback interface
+	RemoveVirtualIP(ip *net.IPNet) error
```

Comment:
Agree, from the sandbox pov this is an alias IP on the loopback interface. I will change to `AddLookbakcAliasIP`. Thanks.